common/src/main/java/com/gentics/mesh/core/link/WebRootLinkReplacerImpl.java

```diff
@@ -11,6 +11,7 @@
 import javax.inject.Inject;
 import javax.inject.Singleton;
 
+import com.gentics.mesh.cli.ODBBootstrapInitializer;
```

Comment:
Check imports:
* Maintain order of imports
* Do not use wildcard imports
* Static imports come first